### PR TITLE
feat: last waterings + report problem

### PR DESCRIPTION
**Pull request by Jaszkowic**
- last watering sections
- tests
- i18n

for testing, use tree `_hwz84dml7` or `_lur9t5w9p` which in production have some recent waterings

### CODE REVIEW

**Comment by Jaszkowic** [comment]
still needs i18n

**Comment by Jaszkowic** [comment]
i18n done. 

please not that this PR goes against https://github.com/technologiestiftung/giessdenkiez-de/pull/728 which should be reviewed first.

File: src/components/tree-detail/watering-card.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  wateringData: TreeWateringData;
+}
+
+const LastWaterings: React.FC<WateringCardProps> = ({ wateringData }) => {
+  return (
+    <div className="shadow-gdk-hard flex flex-col gap-2 rounded-lg p-4">
+      <div className="font-bold">{wateringData.username}</div>
+      <div className="flex flex-row items-center justify-between">
+        <div>
+          {new Date(wateringData.timestamp).toLocaleDateString(undefined, {
```
Comment (raphael-arce) [inline src/components/tree-detail/watering-card.tsx:14]: I think this should be a state function related to i18n. Similar to the `formatNumber()` function we added, let's add a `formatDate()` that uses the current language.

File: src/utils/date-utils.ts
Comment (raphael-arce) [inline src/utils/date-utils.ts:1]: I know we tried to reduce as much as possible the external libraries used, but when working with dates, I think it is very easy to unknowingly overlook edge cases (happened to me a few times!) and I think it's worth to use e. g. https://date-fns.org/ (where many times existing functions are directly available). What do you think?

**Review by raphael-arce — CHANGES_REQUESTED** [review]
I have a couple suggestions, also I couldn't test it on the preview with the given treeIds, let's maybe test it together tomorrow 👍

File: src/utils/date-utils.ts
Comment (Jaszkowic) [reply src/utils/date-utils.ts:1]: i think you are right, much less pain

File: src/components/tree-detail/watering-card.tsx (hunk 1 above)
Comment (Jaszkowic) [reply src/components/tree-detail/watering-card.tsx:14]: good idea

**Pull request merged**